### PR TITLE
Onboard SSO On HyperShift

### PR DESCRIPTION
**Pull request by liswang89**

### CODE REVIEW

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-rosa-hypershift-rhsso-interop-aws

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-rosa-hypershift-rhsso-interop-aws

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-rosa-hypershift-rhsso-interop-aws

**Comment by liswang89** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-rosa-hypershift-rhsso-interop-aws

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml [hunk 1]
```diff
@@ -96,6 +96,87 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-ibmcloud
```
Comment (calebevans) [inline ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml:98]: Need a slack config here

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml [hunk 2]
```diff
@@ -187,17 +268,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-interop-rhsso-interop-ibmcloud
```
Comment (calebevans) [inline ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml:270]: Slack config was removed by `make update` here

File: ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.15-lp-rosa-hypershift.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "RHSSO", "group": {"name": "lp-tests", "priority": 2},"jira_additional_labels": ["rhsso","!default","interop-tests"],"jira_assignee":"ikhomyn@redhat.com"}
+            ],
+          "success_rules":
```
Comment (calebevans) [inline ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.15-lp-rosa-hypershift.yaml:56]: You can remove the "success_rules" here. They are in https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/aws-ipi/lp-interop.json already.

File: ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-rosa-hypershift.yaml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+            [
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "pod_failure", "classification": "Test Execution", "group": {"name": "lp-tests", "priority": 2}, "jira_additional_labels": ["!default","interop-tests"]},
+              {"step": "keycloak-qe-rhsso-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "RHSSO", "group": {"name": "lp-tests", "priority": 2},"jira_additional_labels": ["rhsso","!default","interop-tests"],"jira_assignee":"ikhomyn@redhat.com"}
+            ],
+          "success_rules":
```
Comment (calebevans) [inline ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-rosa-hypershift.yaml:56]: We can remove success rules

**Review by calebevans — CHANGES_REQUESTED** [review]

**Review by calebevans — APPROVED** [review]

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml (hunk 1 above)
Comment (liswang89) [reply ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml]: updated.

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml (hunk 2 above)
Comment (liswang89) [reply ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml]: updated.

File: ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-rosa-hypershift.yaml (hunk 4 above)
Comment (liswang89) [reply ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-rosa-hypershift.yaml]: updated.

**Pull request merged**